src/main/java/com/wego/dao/models/CachedResponse.java

```diff
@@ -0,0 +1,170 @@
+package com.wego.dao.models;
+
+import com.ning.http.client.FluentCaseInsensitiveStringsMap;
+import com.ning.http.client.Response;
+import com.ning.http.client.cookie.Cookie;
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.MalformedURLException;
+import java.net.URI;
+import java.nio.ByteBuffer;
+import java.util.List;
+import org.apache.commons.collections4.MapUtils;
+import org.apache.commons.lang3.StringUtils;
+
+public class CachedResponse implements Response {
+  private String id;
+  private String responseBody;
+  private List<Cookie> cookies;
+  private FluentCaseInsensitiveStringsMap headers;
+  private Integer statusCode;
```

Comment:
oh, okay. can we return it as a null on the getter instead of returning 0 here https://github.com/wego/async-http-cache/pull/1/files#diff-ceec446c2bb5f8d2ba305936fa32ebdeR25?